src/test/scala/viper/gobra/parsing/GoParserUnitTests.scala

```diff
@@ -0,0 +1,197 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
```

Comment:
I would use here the license of the Go tests as that's the valuable part of this file